firmware/controllers/gauges/tachometer.h

```diff
@@ -7,7 +7,9 @@
 
 #pragma once
 
+#if !EFI_UNIT_TEST
```

Comment:
I think this include here should be unconditional regardless of unit tests or not

with master changes I believe no changes to tachometer.h are needed